ci-operator/jobs/openshift-ossm/maistra/openshift-ossm-maistra-postsubmits.yaml

```diff
@@ -0,0 +1,545 @@
+postsubmits:
+  maistra/test-infra:
+  - name: test-infra_push-containers
+    decorate: true
+    path_alias: github.com/maistra/test-infra
+    skip_report: false
+    run_if_changed: '^docker/maistra-builder_.*\.Dockerfile|^docker/scripts'
+    branches:
+      - main
+    labels:
+      preset-quay-pusher: "true"
+    max_concurrency: 1
+    spec:
+      containers:
+      - image: "quay.io/maistra-dev/maistra-builder:2.0"
+        imagePullPolicy: Always
+        command:
+        - entrypoint
+        - make
+        - maistra-builder.push
+        securityContext:
+          privileged: true
+
+  - name: test-infra_push-proxy-containers
+    decorate: true
+    path_alias: github.com/maistra/test-infra
+    skip_report: false
+    run_if_changed: '^docker/maistra-proxy-builder_.*\.Dockerfile'
+    branches:
+      - main
+    labels:
+      preset-quay-pusher: "true"
+    max_concurrency: 1
+    spec:
+      containers:
+      - image: "quay.io/maistra-dev/maistra-builder:2.0"
+        imagePullPolicy: Always
+        command:
+        - entrypoint
+        - make
+        - maistra-proxy-builder.push
+        securityContext:
```

Comment:
Hello, CI maintainer, 
Is there a way of running a privileged pod in an OpenShift CI environment ?
We use that pod for running a dockerd and upstream integration tests. 
Thanks,

Reply:
We do have precedent for doing it like this. Also for adding `runAsUser: 0`. Would that satisfy your needs?